### PR TITLE
MGMT-12553: Adjusting the index_name for queries

### PR DESCRIPTION
**Pull request by danmanor**
Adding a "-*" suffix in order that all jobs indexes will be considered while querying data from elasticsearch

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -116,7 +116,7 @@ parameters:
 - name: ES_STEP_INDEX
   value: "steps"
 - name: ES_JOB_INDEX
-  value: "jobs"
+  value: "jobs-*"
```
Comment (rccrdpccl) [inline openshift/template.yaml:119]: why this change?

**Comment by osherdp** [comment]
> /ok-to-test

I think you no longer need to type this
Try next time to see if jobs are starting without it

**Comment by danmanor** [comment]
ok @osherdp

File: openshift/template.yaml (hunk 1 above)
Comment (osherdp) [reply openshift/template.yaml]: Just to say it explicitly, you should always have as much context as needed in the PR description and commit message instead of saying what you're doing in the PR

For example:
```
Searching `jobs` index does not yield any result, because the created indexes to the jobs are of the pattern: `jobs-<date>`. For example `jobs-2022-11-21`.

With current code we're getting the following result:

Querying elasticsearch with the query ...
Got response: []
```

Comment (rccrdpccl) [reply openshift/template.yaml]: :+1: next time add description to the PR please :)
Also, did you look into [aliases](https://opensearch.org/docs/1.2/opensearch/index-alias/)?
They might be a better fit for this purpose

Comment (adriengentil) [reply openshift/template.yaml]: Interesting! and then you query the alias as an index?

Let's keep the wildcard for the moment, and see if we can leverage an alias later.

**Review by adriengentil — APPROVED** [review]

File: openshift/template.yaml (hunk 1 above)
Comment (danmanor) [reply openshift/template.yaml]: @osherdp ok thanks I thought just making this quick patch in order to make it work, should have been more explict

Comment (osherdp) [reply openshift/template.yaml]: No worries
Just try for the next changes to be more explicit (and include the description also in the commit message itself)
